### PR TITLE
new aegir

### PR DESCRIPTION
**Pull request by daviddias**
@dignifiedquire I'm seeing the same errors I'm getting in https://github.com/ipfs/js-ipfs-api/pull/585#issuecomment-325188935 on this test repo with the latest aegir.

Did you get to test the new aegir before merge and release? This repo should always be tested before shipping a new version as it is a very solid way to ensure that it works with our projects.

### CODE REVIEW

**Comment by dignifiedquire** [comment]
things were passing for me last time I checked

**Comment by daviddias** [comment]
@dignifiedquire is this something you know what's going on or that requires more debugging?

**Comment by dignifiedquire** [comment]
@diasdavid need probably the old timeouts of `80000` back, given how slow these tests and CI are

**Comment by daviddias** [comment]
@dignifiedquire still failing

**Comment by dignifiedquire** [comment]
Those failures look like issues with the tests/code rather than aegir

**Comment by daviddias** [comment]
@dignifiedquire what surprises me is that aegir 12.0.0 got merged and released without having it fully passing in this repo. This might mean that JEST is not suited yet to our type of testing. 

If there are bugs with the tests lets fix it, however, we should not lightly release aegir without testing it extensively in this repo.

**Comment by dignifiedquire** [comment]
I was not under the impression that releasing a version of aegir has the requirement of this repo passing on that version. For me that was simply a nice way to test certain things. 

The current version working well on a number of repos, the test on js-ipfs and js-ipfs-api have been very brittle for some time, so while it is very possible there is an issue in aegir and its components I am not convinced that should have held up the release making it harder and more problematic to test.

**Comment by daviddias** [comment]
> I was not under the impression that releasing a version of aegir has the requirement of this repo passing on that version. For me that was simply a nice way to test certain things.

Ok. It is a great test to ensure that new versions of aegir work well for our set up of tests. That is why this repo is a clone of js-ipfs-api because it has one of the most complicated and complete test setups we have built.

Let's get everything working :) Happy to help.

**Comment by dignifiedquire** [comment]
The latest version of this branch together with https://github.com/ipfs/interface-ipfs-core/pull/151  passes all node test on my local machine

**Comment by dignifiedquire** [comment]
update all browser tests just passed locally as well

File: package.json [hunk 1]
```diff
@@ -10,16 +10,15 @@
     "http": "stream-http"
   },
   "scripts": {
-    "test": "aegir test",
-    "test:node": "aegir test --target node",
-    "test:browser": "aegir test --target browser webworker",
+    "test": "aegir test --no-parallel --timeout 80000",
```
Comment (daviddias) [inline package.json:13]: Which tests still require increased timeout?

Comment (dignifiedquire) [reply package.json:13]: not sure..ci was just being slow and I didn't want to spend an hour on going test by test on ci

**Comment by dignifiedquire** [comment]
and webworker tests passed as well

**Comment by daviddias** [comment]
As part of testing aegir with this repo, every release must get every single feature tested, from testing, to coverage to release. Right now https://github.com/ipfs/js-ipfs-api/pull/585 is being super painful and there is incoming changes to interface-ipfs-core that I'll have to release in patch versions of the previous minor so that it works with old aegir if we don't get new aegir to work.

**Comment by daviddias** [comment]
solved with https://github.com/ipfs/aegir-test-repo/pull/3

**Pull request closed without merging**